crates/pop-cli/src/commands/mod.rs

```diff
@@ -53,6 +52,16 @@ fn about_build() -> &'static str {
 	return "Build a smart contract or Rust package.";
 }
 
+/// Help message for the up command.
+fn about_up() -> &'static str {
+	#[cfg(all(feature = "parachain", feature = "contract"))]
+	return "Deploy a smart contract or launch a local network.";
+	#[cfg(all(feature = "parachain", not(feature = "contract")))]
+	return "Launch a local network.";
+	#[cfg(all(feature = "contract", not(feature = "parachain")))]
```

Comment:
```suggestion
	#[cfg(all(not(feature = "parachain"), feature = "parachain"))]
```

I prefer this way to keep the feature order consistent to the above.

Reply:
Is a valid point! Is there a best practice in Rust for this? 
I followed the same format as [about_build()](https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-cli/src/commands/mod.rs) above, so if we change it here, we should also update `about_build()` for consistency.

Reply:
I don't find any document for the best practices but GPT recommends :) : "When specifying multiple conditions, it is best to keep a consistent order for readability. The order in `all(...)` or `any(...)` does not affect behavior, but consistency makes it easier to scan."

Reply:
Noted for a future refactor. I am keeping it as is for consistency with the rest of the code in this PR.